### PR TITLE
Coin Tower - DP

### PR DESCRIPTION
**Pull request by ishwari20**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Jack and Harry are playing a new game today. They form a tower of N coins and make a move in alternate turns. harry plays first. In one step, the player can remove either 1, X, or Y coins from the tower. The person to make the last move wins the game. Find who wins the game for different number of coins.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #7323

### CODE REVIEW

File: C-Plus-Plus/dp/coinTower.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        return "Jack";
+    }
+}
+
+int main()
+{
+	int n, x, y;
+	cin >> n >> x >> y;
+	cout << findWinner(n, x, y);
+}
```
Comment (siddharth25pandey) [inline C-Plus-Plus/dp/coinTower.cpp]: Please write messages in Display form for  what are you taking input from the user or what are you showing output to the user for better User Interface

File: C-Plus-Plus/dp/coinTower.cpp [hunk 2]
```diff
@@ -0,0 +1,97 @@
+/*
+Jack and Harry are playing a new game today. They form a tower of N coins 
+and make a move in alternate turns. harry plays first. In one step, the player 
+can remove either 1, X, or Y coins from the tower. The person to make the last 
+move wins the game. Find who wins the game for different number of coins.
+*/
+
+
+#include <bits/stdc++.h>
+#include <string>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/dp/coinTower.cpp]: No need to add string header file after adding bits header (Redundant Line)
Remove it

File: C-Plus-Plus/dp/coinTower.cpp [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    }
+    else{
+        return "Jack";
+    }
+}
+
+int main()
+{
+	int n, x, y;
+	cin >> n >> x >> y;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/dp/coinTower.cpp]: Use good naming convention for the variables

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@ishwari20 Do the suggested changes

**Comment by ishwari20** [comment]
@siddharth25pandey I have made those changes, please review

**Review by siddharth25pandey — APPROVED** [review]
@ishwari20 Perfectly explained, Bingo 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @ishwari20 for her code

**Pull request merged**